app/assess/models/ui/applicants_response.py

```diff
@@ -279,6 +311,11 @@ def _ui_component_from_factory(item: dict, application_id: str):
             item["answer"] = "£{:.2f}".format(float(item["answer"]))
         return BesideQuestionAnswerPair.from_dict(item)
 
+    elif presentation_type == "integer":
+        if item.get("answer"):
+            item["answer"] = "{}".format(int(item["answer"]))
+        return BesideQuestionAnswerPair.from_dict(item)
```

Comment:
Might be good to add an entry to this parametrized test for some coverage of this:

https://github.com/communitiesuk/funding-service-design-assessment/blob/8becb0c74c53a86482a54b66ee11e874c68e22f6/tests/test_applicators_response.py#L273-L281

Reply:
Nice suggestion. Done now